src/components/Kv/KvButton.vue

```diff
@@ -124,6 +144,15 @@ export default {
 		text-decoration: $anchor-text-decoration-hover;
 		background: transparent;
 	}
+
+	&--red {
```

Comment:
I don't feel strongly, but should this be `&--alert`?  In a future version of KvButton, it'd be nice to have all these classes set via props, e.g., `<kv-button size="large" variant="primary">` or `<kv-button size="text" variant="alert">`. This would let us mix and match easier.